sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_get_manual_validation_info_async.py

```diff
@@ -0,0 +1,97 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_get_manual_validation_info_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to output the information that will help with manually
+    validating your output from recognize custom forms.
+USAGE:
+    python sample_get_manual_validation_info_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+    3) CUSTOM_TRAINED_MODEL_ID - the ID of your custom trained model
+"""
+
+import os
+import asyncio
+from pathlib import Path
+
+
+class GetValidationInfoSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+    model_id = os.environ["CUSTOM_TRAINED_MODEL_ID"]
+
+    async def get_manual_validation_info_from_recognize_custom_forms(self):
+        # the sample forms are located in this file's parent's parent's files.
+        path_to_sample_forms = Path(__file__).parent.parent.absolute() / Path("sample_forms/forms/Form_1.jpg")
+        # [START create_form_recognizer_client_async]
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.formrecognizer.aio import FormRecognizerClient
+
+        form_recognizer_client = FormRecognizerClient(
+            endpoint=self.endpoint, credential=AzureKeyCredential(self.key)
+        )
+        # [END create_form_recognizer_client_async]
+        async with form_recognizer_client:
+
+            # The form you are recognizing must be of the same type as the forms the custom model was trained on
```

Comment:
This isn't strictly speaking true ... we've found examples of models trained on one form type that will extract field values for fields with the same form field label from other forms.  A model can determine what type of form a model is, if it supports multiple form types.

Reply:
Ok, I'll get rid of references to this

Reply:
My understanding is that you should only recognize forms whose form type is included int he list of form types the custom model can recognize, is this not true?

I can also try wording this more clearly. How about "Make sure your form type is included in the form types the custom model can recognize"?

Reply:
Yes, I think that works better.  Thanks!